Bugfix: compound v2, add check of the event address in mint and borrow logic (#7878)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :bug:`-` compound_v2 decoder will now check that the event address is a compound one for the borrow and mint event decoding. This fixes a bug with decoding transactions containing also flash loans of same asset type.

Reviewer: This is a developer facing changelog message. Adjust it.

```suggestion
* :bug:`-` Compound v2 transactions containing also flash loans of same asset type will now be properly decoded.
```
